phpBB/includes/functions_content.php

```diff
@@ -1758,3 +1758,48 @@ function merge($bitfield)
 		$this->data = $this->data | $bitfield->get_blob();
 	}
 }
+
+/**
+ * Formats the quote according to the given BBCode status setting
+ *
+ * @param bool 						$bbcode_status The status of the BBCode setting
+ * @param array 					$quote_attributes The attributes of the quoted post
+ * @param phpbb\textformatter\utils $text_formatter_utils Text formatter utilities
+ * @param parse_message 			$message_parser Message parser class
+ * @param string 					$message_link Link of the original quoted post
+ * @since 3.2.4-RC1
+ */
+function format_quote($bbcode_status, $quote_attributes, $text_formatter_utils, $message_parser, $message_link = '')
```

Comment:
Please change the name to `phpbb_format_quote()`. This is done on new functions to reduce the possibility of conflicts with other functions outside of phpBB.